### PR TITLE
Improvements to cadcetrans

### PR DESCRIPTION
**Pull request by andamian**

### CODE REVIEW

File: cadcdata/cadcdata/core.py [hunk 1]
```diff
@@ -628,6 +634,13 @@ def main_app():
         '-s', '--archive-stream',
         help='specific archive stream to add the file to',
         required=False)
+    put_parser.add_argument(
+        '-i', '--input',
+        help='space-separasted list of input name to use in archive - '
```
Comment (at88mph) [inline cadcdata/cadcdata/core.py]: separated

File: cadcdata/cadcdata/core.py [hunk 2]
```diff
@@ -781,11 +796,26 @@ def handle_error(msg, exit_after=True):
...
                 handle_error('No files found to put')
-            for f in files:
-                client.put_file(archive, f, archive_stream=args.archive_stream,
-                                mime_type=args.type,
-                                mime_encoding=args.encoding,
-                                md5_check=(not args.nomd5))
+            if args.input:
+                input_names = args.input.split()
+                if len(input_names) != len(files):
+                    handle_error('The input names does not match number of '
```
Comment (at88mph) [inline cadcdata/cadcdata/core.py]: Number of input names does not match...

File: cadcdata/cadcdata/tests/data/help_put.txt [hunk 3]
```diff
@@ -22,6 +23,9 @@ optional arguments:
   -h, --help            show this help message and exit
   --host HOST           base hostname for services - used mainly for testing
                         (default: www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca)
+  -i, --input INPUT     space-separasted list of input name to use in archive
```
Comment (at88mph) [inline cadcdata/cadcdata/tests/data/help_put.txt]: separated

File: cadcetrans/README.rst [hunk 4]
```diff
@@ -5,5 +5,80 @@ cadcetrans
...
+
+    cd docker
+    docker build -t cadcetrans .
+
+To invoke it:
+
+::
+
+    docker run --rm --mount type=bind,source=/tmp/logs,target=/logging
+    --mount type=bind,source=/tmp/input/,target=/input cadcetrans status
```
Comment (at88mph) [inline cadcetrans/README.rst]: The `--mount` stuff is old.  Should read:
```
docker run --rm -v /tmp/logs:/logging -v /tmp/input:/input cadcetrans status
```

File: cadcetrans/README.rst [hunk 5]
```diff
@@ -5,5 +5,80 @@ cadcetrans
...
+
+or:
+
+::
+
+    docker run --rm --mount type=bind,source=/tmp/logs,target=/logging
+    --mount type=bind,source=/tmp/input/,target=/input cadcetrans data
```
Comment (at88mph) [inline cadcetrans/README.rst]: Ditto for `--mount`.

File: cadcetrans/cadcetrans/etrans_core.py [hunk 6]
```diff
@@ -733,7 +728,10 @@ def handle_error(msg, exit_after=True):
     elif args.debug:
         logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
     else:
-        logging.basicConfig(level=logging.WARN, stream=sys.stdout)
+        if (args.cmd != 'status') and args.dryrun:
+            logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+        else:
+            logging.basicConfig(level=logging.WARN, stream=sys.stdout)
```
Comment (at88mph) [inline cadcetrans/cadcetrans/etrans_core.py]: Shouldn't warnings go to `sys.stderr`?

File: cadcdata/cadcdata/core.py (hunk 1 above)
Comment (andamian) [reply cadcdata/cadcdata/core.py]: Done

**Pull request merged**